client/src/components/page/Home.js

```diff
@@ -17,44 +17,144 @@
 import React, { Component } from 'react';
 import CustomMap from 'components/ui/CustomMap.js';
 import HOME_PAGE_MARKERS from 'components/markers/MarkersData.js';
+import { withStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import background_pasta from 'statics/images/food_icon.jpg';
+import Avatar from '@material-ui/core/Avatar';
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import TextField from '@material-ui/core/TextField';
+import Link from '@material-ui/core/Link';
+import Paper from '@material-ui/core/Paper';
+import Grid from '@material-ui/core/Grid';
+import LockOutlinedIcon from '@material-ui/icons/LockOutlined';
+import PropTypes from 'prop-types';
 
 const GOOGLE_MAPS_API_URL =
   'https://maps.googleapis.com/maps/api/js?key=AIzaSyAi9TMtkY74gzfmjPkD7w1Tu-zyABHYlww&v=3.exp&libraries=geometry,drawing,places';
 const DEFAULT_MAP_ZOOM = 15;
 const GOOGLEPLEX_COORD = { lat: 37.422, lng: -122.084 };
 
+/**  const theme = createMuiTheme({
+  palette: { primary: { main: grey[900] } },
+  typography: {
+    fontSize: 20,
+    fontWeight: 1200
+  }
+}); */
+const styles = function() {
+  return {
+    root: {
+      height: '100vh'
+    },
+    image: {
+      backgroundImage: `url(${background_pasta})`,
+      backgroundRepeat: 'no-repeat',
+      backgroundSize: 'cover',
+      backgroundPosition: 'center',
+      alignItems: 'center'
+    },
+    paper: {
+      margin: 24,
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center'
+    },
+    avatar: {
+      margin: 24
+      //backgroundColor: theme.palette.secondary.main
+    },
+    form: {
+      width: '100%', // Fix IE 11 issue.
```

Comment:
what is the IE 11 issue?

Reply:
That was left over from copying and pasting the code from an example online